core/domain/base_model_validators.py

```diff
@@ -90,6 +93,44 @@ def __init__(
         self.model_ids = model_ids
 
 
+class UserSettingsModelFetcherDetails(python_utils.OBJECT):
+    """Value object providing the class and ids to fetch the
+    user settings model.
+    """
+
+    def __init__(
+            self, field_name, model_class, model_ids,
```

Comment:
`model_class` is not needed right? It will always be `UserSettingsModel` right?

Reply:
Done.